SampleGallery/Samples/SDK 17134/RadialGradients/RadialGradients.xaml.cs

```diff
@@ -0,0 +1,310 @@
+
+                _PBGradientStop2.StartAnimation("Offset", _stop2offsetAnim2);
+
+                // When the stopwatch starts, the color for the gradient stops are changed to Alice Blue so that the pulse is visible
+                _pulseColor = _compositor.CreateColorKeyFrameAnimation();
+                _pulseColor.InsertKeyFrame(0, _transparent);
+                _pulseColor.InsertKeyFrame(.99f, _aliceBlue);
+                _pulseColor.InsertKeyFrame(1, _transparent);
+                _pulseColor.Duration = TimeSpan.FromSeconds(1);
+                _pulseColor.IterationBehavior = AnimationIterationBehavior.Forever;
+
+                _PBGradientStop1.StartAnimation(nameof(_PBGradientStop1.Color), _pulseColor);
+
+                // Create the animation that animates the scale of the pulsing sprite visual
+                _scale = _compositor.CreateVector3KeyFrameAnimation();
+                _scale.InsertKeyFrame(0, new Vector3(0, 0, 0));
+                _scale.InsertKeyFrame(1, new Vector3(1, 1, 0));
+                _scale.Duration = TimeSpan.FromSeconds(1);
+                _scale.IterationCount = 50;
+
+                _pulseVisual.StartAnimation("Scale", _scale);
+
+                // Set up the stopwatch to reset each it is started again
+                _dt = new DispatcherTimer();
+                _stopwatch = new Stopwatch();
+                _dt.Interval = TimeSpan.FromMilliseconds(0.00f);
+                _dt.Tick += Dt_Tick;
+                _dt.Start();
+                _stopwatch.Start();
+
+                // Set boolean flag for if the stopwatch animation is on to true
+                animationOn = true;
+            }
+            else
+            {
+                // Button turns the colors of it's gradient stops back to cooler colors when it is shut off
+                _changeButtonGradientStop1 = _compositor.CreateColorKeyFrameAnimation();
+                _changeButtonGradientStop1.InsertKeyFrame(0, _indianRed);
+                _changeButtonGradientStop1.InsertKeyFrame(1, _blueViolet);
+                _changeButtonGradientStop1.Duration = TimeSpan.FromSeconds(2);
+
+                _BBGradientStop1.StartAnimation(nameof(_BBGradientStop1.Color), _changeButtonGradientStop1);
+
+                _changeButtonGradientStop2 = _compositor.CreateColorKeyFrameAnimation();
+                _changeButtonGradientStop2.InsertKeyFrame(0, _orangeRed);
+                _changeButtonGradientStop2.InsertKeyFrame(1, _lightBlue);
+                _changeButtonGradientStop2.Duration = TimeSpan.FromSeconds(2);
+
+                _BBGradientStop2.StartAnimation(nameof(_BBGradientStop2.Color), _changeButtonGradientStop2);
+
+                // Animations for the pulsing effect are turned off
+                _stop1OffsetAnim = _compositor.CreateScalarKeyFrameAnimation();
+                _stop1OffsetAnim.InsertKeyFrame(0, 0);
+                _stop1OffsetAnim.InsertKeyFrame(0f, 0f);
```

Comment:
The f suffix is redundant here. Also you are inserting the 0,0 keyframe twice which I'm pretty sure you don't mean to.